packages/runtime/src/webcontainer/terminal-config.ts

```diff
@@ -59,9 +66,10 @@ export class TerminalPanel implements ITerminal {
 
   constructor(
     readonly type: TerminalPanelType,
-    name?: string,
-    id?: string,
+    private readonly options?: TerminalPanelOptions,
   ) {
+    let name = options?.name;
```

Comment:
Wondering if we should have named this `title` like for `previews`.

Reply:
Oh yeah, I agree. Let's change it?